core/src/test/java/org/infinispan/protostream/ProtobufUtilTest.java

```diff
@@ -226,6 +227,28 @@ public void testJsonWithDifferentFieldOrder() throws Exception {
       assertEquals(createAccount(), account);
    }
 
+   @Test
+   public void vectorFieldsTranscoding() throws Exception {
```

Comment:
`testJsonBytesWithIntArray`?

Reply:
`testJsonBytesWithIntArray` is definitely a better name, thank you!